lowes/lvfs
# Added tests for URL methods in Local Mode

## Objective
To start building out a testing framework for LVFS.

## To Test
Run `PYTHONPATH=/path/to/lvfs pytest /path/to/lvfs/tests`. 
You should see something like this: 
<img width="1279" alt="Screen Shot 2021-11-01 at 9 28 59 AM" src="https://user-images.githubusercontent.com/10490006/139679339-b736de72-d9ca-4d60-8428-df7ce32ee2e2.png">

## Commits
-  (25ddbf7) added tests for local URL usage
